pkgs/development/tools/ytt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ pkgs, lib, ... }:
+with pkgs; with lib; buildGoModule rec {
```

Comment:
```suggestion
{  lib }:
buildGoModule rec {
```

Don't use with for the entire scope. Also don't pass in pkgs. Triple dots are only needed if you can add any number of extra packages which is not needed here.

Reply:
Thanks for the suggestions. I'll definitely remove the generous use of with, but unless I'm missing something, I'll need to keep `pkgs` as it's required for `buildGoModule`.

Reply:
I've figured this out: I can just put required functions in this function's set and it'll work out.